FRBDK/Glue/Glue/MVVM/CommandBase.cs

```diff
@@ -22,6 +22,21 @@ public CommandBase(Action execute, Func<bool> canExecute = null)
             this.canExecuteFunc = canExecute;
         }
 
+        public CommandBase(Action execute, string canExecuteProperty, INotifyPropertyChanged viewModel)
+        {
+            this.execute = execute;
+
+            this.canExecuteFunc = () => viewModel?.GetType().GetProperty(canExecuteProperty, BindingFlags.Instance | BindingFlags.NonPublic | BindingFlags.Public)?.GetValue(viewModel) is true;
```

Comment:
I'm not a fan of this implementation and similar ones in the new generic version, but it does the job without pulling in third party libraries.